Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM quay.io/ukhomeofficedigital/openjdk8:v1.8.0.171
+
+USER root
+
+# Copy Test Repository into the Docker Image
+COPY . /app
+
+# Create gradle user
+ENV USER user-gradle
+ENV USER_ID 1001
+ENV GROUP group-gradle
+
+# Set gradle variables
+ENV GRADLE_VERSION 4.6
+ENV GRADLE_HOME /usr/local/gradle
+ENV PATH ${PATH}:${GRADLE_HOME}/bin
+ENV GRADLE_USER_HOME /home/${USER}
+
+RUN yum install wget unzip -y -q
+
+# Install gradle
+WORKDIR /usr/local
+RUN wget  https://services.gradle.org/distributions/gradle-$GRADLE_VERSION-bin.zip -q && \
+    unzip -q gradle-$GRADLE_VERSION-bin.zip && \
+    rm -f gradle-$GRADLE_VERSION-bin.zip && \
+    ln -s gradle-$GRADLE_VERSION gradle
```

Comment:
Why are we installing gradle instead of just copying gradlew from the project?

Reply:
I shamelessly copied this from the E2E tests. What you say makes sense so I'll look into it.

Reply:
As discussed, it transpires that using the wrapper means the first thing the running container does is download and install gradle. That takes time and adds a dependency on the availability of the Gradle download. I've commented the Dockerfile with this reasoning.